core/templates/components/forms/custom-forms-directives/edit-thumbnail-modal.component.ts

```diff
@@ -115,17 +113,21 @@ export class EditThumbnailModalComponent {
       this.imgSrc = reader.result as string;
       this.updateBackgroundColor(this.tempBgColor);
       img.src = this.imgSrc;
-      this.uploadedImage = this.imgSrc;
+      let svg = this.svgSanitizerService.parseDataURI(this.imgSrc);
       this.invalidTagsAndAttributes = (
-        this.svgSanitizerService.getInvalidSvgTagsAndAttrsFromDataUri(
-          this.imgSrc));
-      this.tags = this.invalidTagsAndAttributes.tags;
-      this.attrs = this.invalidTagsAndAttributes.attrs;
-      if (this.tags.length > 0 || this.attrs.length > 0) {
-        this.reset();
-      } else {
-        this.thumbnailHasChanged = true;
-      }
+        this.svgSanitizerService.getInvalidSvgTagsAndAttrs(svg));
+      const tags = this.invalidTagsAndAttributes.tags;
+      let attrs: string[] = [];
+      this.invalidTagsAndAttributes.attrs.forEach(attribute => {
```

Comment:
Add a comment above this line to explain what the code is doing. It's fine if the comment is long, as long as it gives the reader a clear understanding of what is going on.

Reply:
This part of the code has been removed now.